styles/main.css

```diff
@@ -34,8 +34,6 @@ h3 {
 **********/
 
 footer {
-    position: absolute;
```

Comment:
Got in the way. Should be solved by flex box (http://philipwalton.github.io/solved-by-flexbox/) anyway.